plotly/tools.py

```diff
@@ -2332,7 +2332,7 @@ def create_candlestick(open, high, low, close,
 
     @staticmethod
     def create_distplot(hist_data, group_labels,
-                        bin_size=1., curve_type='kde',
+                        bin_size=[1.], curve_type='kde',
```

Comment:
On second thought, why not just leave it as `1.`? We need to be backwards compatible anyhow.